kati-frantz/more-recipes
# Feature/client/single recipe data

#### What does this PR do

- Add a complete version (kinda) of the single recipe view

#### Description of Task to be completed

- Implement single recipe display
- Implement reviews for recipe display
- Implement create review for recipe

#### How should this be manually tested

- Start dev client and backend server
- Visit client in browser
- Create a recipe
- You are automatically redirected to the page for that recipe
- Review a recipe
- You automatically see your review
- Make more reviews. You see reviews in scroll bar

### Any background context you want to provide ? 

N/A

### What are the relevant pivotal tracker stories ? 

152164686

### Screenshots
N/A
### Questions:
N/A
